src/validation.cpp

```diff
@@ -2680,7 +2682,8 @@ bool Chainstate::ConnectBlock(const CBlock& block, BlockValidationState& state,
                               tx_state.GetRejectReason(), tx_state.GetDebugMessage());
                 break;
             }
-            control.Add(std::move(vChecks));
+            if (control)
+                control->Add(std::move(vChecks));
```

Comment:
In commit "validation: only create a CCheckQueueControl if it's actually going to be used" (bcde9153012b2ce3e9ff5b7cd1c6f9866453e727)

Developer notes want single-line if or multiple line if with braces, not multiline if without braces.

>  - If an `if` only has a single-statement `then`-clause, it can appear
>    on the same line as the `if`, without braces. In every other case,
>    braces are required, and the `then` and `else` clauses must appear
>    correctly indented on a new line.